src/js/common/components/LoginModal.jsx

```diff
@@ -54,10 +72,10 @@ class LoginModal extends React.Component {
 
   handleLogin(e) {
     e.preventDefault(); // Don't try to submit the page
+    this.setState({ loginButtonClicked: true });
     const loginUrl = this.props.user.login.loginUrl;
```

Comment:
Good point. I forget about the ability to rename in deconstruction statements like this. :man_shrugging: